lib/rmt/downloader.rb

```diff
@@ -85,10 +85,14 @@ def verify_checksum(filename, checksum_type, checksum_value)
 
   def make_request(remote_file, local_file, request_fiber, checksum_type = nil, checksum_value = nil)
     uri = URI.join(@repository_url, remote_file)
-    uri.query = @auth_token if @auth_token
+    uri.query = @auth_token if (@auth_token && uri.scheme != 'file')
```

Comment:
no parens necessary

Reply:
@lagartoflojo Rubocop doesn't complain though!
Everything's nicer with parens. And semicolons :D